Sources/Sentry/SentryClient.m

```diff
@@ -159,7 +160,8 @@ - (SentryId *)captureError:(NSError *)error
 - (SentryEvent *)buildErrorEvent:(NSError *)error
 {
     SentryEvent *event = [[SentryEvent alloc] initWithLevel:kSentryLevelError];
-    event.message = [NSString stringWithFormat:@"%@ %ld", error.domain, (long)error.code];
+    NSString *message = [NSString stringWithFormat:@"%@ %ld", error.domain, (long)error.code];
+    event.message = [SentryMessage messageWithFormatted:message];
```

Comment:
L: wondering if here would be a good case to use the params instead of the formatted field.

Reply:
That's very good input thanks 👏 . I will take a look.

Reply:
the downside is that every message will be grouped independently of the domain and error code, not sure if its a good idea, but attach stack traces are enabled by default so it won't group by messages unless it's disabled, right?

Reply:
It is anyway going to be grouped into the same issue if just the error code is different. The advantage of this approach is that you see the different error codes in the message section of the issue. With the previous version, you just saw the error code of one issue and you didn't see if it changed.

Reply:
nice, thanks!